satellite/satellitedb/containment.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"database/sql"
+
+	"storj.io/storj/pkg/audit"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+type containment struct {
+	db *dbx.DB
+}
+
+// Get gets the pending audit by node id
+func (containment *containment) Get(ctx context.Context, id pb.NodeID) (*audit.PendingAudit, error) {
+	if id.IsZero() {
+		return &audit.PendingAudit{}, audit.ContainError.New("node ID empty")
```

Comment:
Just a question. What's the benefit of returning a pointer here vs returning a `nil`?

Reply:
oh good catch! earlier I thought it was erroring because it wanted a pointer returned and not nil but it must've been erroring about something else lol. thanks!